Documentation/kube-flannel-aliyun.yml

```diff
@@ -93,7 +94,7 @@ spec:
       serviceAccountName: flannel
       initContainers:
       - name: install-cni
-        image: registry.cn-hangzhou.aliyuncs.com/google-containers/flannel:v0.9.0
+        image: registry.cn-hangzhou.aliyuncs.com/google-containers/flannel:v0.9.1
```

Comment:
Does this image actually exist? I don't have permissions to push to this repo...

Reply:
Me neither, I thought you already did so. I'll revert this change.